modules/reference/pages/rpk/rpk-topic/rpk-topic-consume.adoc

```diff
@@ -94,6 +94,12 @@ Headers are formatted with percent encoding inside of the modifier:
 %h{%k=%v{hex}}
 ```
 
+=== Values
```

Comment:
This sub-section seems to have been placed in between the `Headers` sub-section. It needs to be moved below the paragraph:

```
This prints all headers with a space before the key and after the value, an equals sign between the key and value, and with the value hex encoded. Header formatting actually just parses the internal format as a record format, so all of the above rules about %K, %V, text, and numbers apply.
```

Reply:
Done! I've reorganized the headings too for better visibility.